hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -58,6 +59,7 @@
 	private static final Logger log = LogManager.getLogger(StateView.class);
 
 	private static final byte[] EMPTY_BYTES = new byte[0];
+	private static final int RETRY_TIMES = 5;
```

Comment:
Should the retries be 0 by default and should be dynamically configured by the test? So, that it doesn't affect any other places

Reply:
As matter of fact, I think all the queries to the binary object store should add the retry mechanism. 
Right now, only File query services run into this issue recently, but I have no doubt the same issue can happen to others given the right triggering condition.  For now,  let's just assume it only happens to File queries.

Right now, the re-try is only limited to file queries. Yeah, I will add this to properties to make it configurable.